Dockerfile

```diff
@@ -23,6 +23,8 @@ RUN git clone https://github.com/openlink/virtuoso-opensource.git \
         && cd .. \
         && rm -r /virtuoso-opensource
 
+HEALTHCHECK CMD wget -q -O - http://localhost:8890/sparql'?query=ASK%20%7B%7D' || exit 1
```

Comment:
@nvdk actually it is hard to implement: to do it properly I need to start a child process that will insert the triples to the database after I did the exec.